web/src/engine/osk/src/specialCharacters.ts

```diff
@@ -18,6 +18,7 @@ let specialCharacters = {
   '*Shifted*':  9,
   '*AltGr*':    2,
   '*TabLeft*':  7,
+  '*BkSp1*':    0x21, //outline shape denoting backspace, *BkSp* is alternate filled shape
```

Comment:
Thanks for your submission! Could you move this block between lines 48-49?
That's the section of new characters added in Keyman 17.0.

At some point, we'd also want to update Developer to add this character. (Similar to #9846)

Reply:
It would be also nice to update the test keyboard
web/src/test/manual/web/issue9469/test9469/source/test9469.keyman-touch-layout

and add a key with text `*BkSp1*` to test the new backspace keycap.